test/JDBC/input/storedProcedures/Test-sp_addrole-vu-verify.sql

```diff
@@ -0,0 +1,11 @@
+-- tsql
+-- Throws an error when role exists
+Exec sp_addrole 'sp_addrole_r1';
+GO
+
+EXEC sp_addrole 'sp_addrole_r2';
+GO
+
+-- Throws an error when role exists
+EXEC sp_addrole 'sp_addrole_r2';
+GO
```

Comment:
Please also select from Babelfish catalog for user/role `sys.babelfish_user_ext`, this is to double check that the created role actually exists and is correctly stored in BBF catalog. There is some example code in `test/JDBC/input/BABEL-ROLE-vu-verify.mix`.
Same applies for drop role